cmd/openshift-install/create.go

```diff
@@ -230,6 +238,26 @@ func newCreateCmd() *cobra.Command {
 	return cmd
 }
 
+func isClusterCreationError(err error) bool {
+	for err != nil {
+		if err.Error() == asset.ClusterCreationError {
+			return true
+		}
+		err = errors.Unwrap(err)
+	}
+	return false
+}
+
+func isConfigurationError(err error) bool {
```

Comment:
It would be easy to create a common function that handles both of these and you pass in either `asset.InstallConfigError` or `assetClusterCreationError`